__tests__/sources.ts

```diff
@@ -695,7 +776,7 @@ query Source($id: ID!) {
       .getRepository(SourceMember)
       .update({ userId: '1' }, { role: SourceMemberRoles.Blocked });
     const res = await client.query(QUERY, { variables: { id: 'a' } });
-    expect(res.data).toMatchSnapshot();
```

Comment:
As seen from the snapshot, there was actually some data to be returned, but it was actually a false return. When a source has members, then it is definitely a Squad. Now that we turned the Squad requested here, the query returned the expected behavior which is not to return anything since you are blocked from the Squad.